onedal/tests/utils/_dataframes_support.py

```diff
@@ -62,6 +65,8 @@ def _as_numpy(obj, *args, **kwargs):
         return obj.asnumpy(*args, **kwargs)
     if dpctl_available and isinstance(obj, dpt.usm_ndarray):
         return dpt.to_numpy(obj, *args, **kwargs)
+    if isinstance(obj, pd.DataFrame) or isinstance(obj, pd.Series):
+        return obj.to_array(*args, **kwargs)
```

Comment:
Like we discussed, this is not touched now, but probably will be used in future pandas output support